src/tasks/task2.js

```diff
@@ -0,0 +1,17 @@
+function min(a, b) {
+

```

Comment:
where is check types of `a` and `b`. use `typeof` operator and validate it that this params are numbers